Jamfile

```diff
@@ -0,0 +1,17 @@
+# Boost.uBLAS
```

Comment:
Actually, I'm not convinced this file should be here. (Sorry to first asking you to rename it !)
The present logic assumes that running `./b2` on the project should build tensor tests and tensor examples, which probably was a nice convenience for you during development, but doens't make much sense now.
The `test/` and `examples/` subdirectories are better integration point to inject the specific build logic for tests and examples respectively.

Reply:
The Boost Build [Documentation](https://boostorg.github.io/build/manual/develop/index.html#bbv2.tutorial.hierarchy) tells something different. We need to clarify this point.

Reply:
See https://github.com/boostorg/gil/blob/develop/.ci/build-and-test.sh#L37-L40 for an example where the test logic iterates over specific (extension) test subdirectories explicitly. We should do the same here.

Reply:
And here is an example of how to conditionally include a target when the `cxxstd` property has a specific value: https://github.com/boostorg/poly_collection/blob/develop/example/Jamfile.v2#L14-L17 (That technique can be used to conditionalize an entire project, too, so I'd suggest you define a project for the entire test/tensors/ subdirectory, as I have done in https://github.com/boostorg/ublas/blob/develop/test/opencl/Jamfile

Reply:
I changed everything. However, e.g. GIL has also a Jamfile at the root directory. I am wondering why we don't have https://github.com/boostorg/gil/blob/develop/Jamfile

Reply:
Well, in the case of GIL, it seems we use that toplevel Jamfile to indicate common compiler flags to be used in all sub-projects. The only other effect that Jamfile has is to allow you to run `b2 libs/gil` as an alias to `b2 libs/gil/test`.
In our case here we have more than tests, so the question is what you'd expect `b2 libs/numeric/ublas` to do ? Build everything, i.e. tests, benchmarks, examples ? Only tests ? It's at best ambiguous, so I'd opt for being explicit, i.e. requiring users to decide themselves what they want to build (and run).

Reply:
okay leaving this issue to be solved on the forked repo.

Reply:
okay removed.